test/unit/preload-test.js

```diff
@@ -0,0 +1,40 @@
+import { getPreload } from 'utils/preload';
+
+describe('getPreload', function() {
+
+    it('should default to metadata when no preload value is given', function() {
+        const preload = getPreload();
+
+        expect(preload).to.equal('metadata');
+    });
+
+    it('should return the primary value when it is valid', function() {
+        const preload = getPreload('auto');
+
+        expect(preload).to.equal('auto');
+    });
+
+    it('should default to metadata when primary value is invalid', function() {
+        const preload = getPreload('aut');
+
+        expect(preload).to.equal('metadata');
+    });
+
```

Comment:
You don't have a test for an invalid fallback value (only for unspecified fallback).